symfony/plugins/orangehrmAdminPlugin/entity/Country.php

```diff
@@ -1,4 +1,21 @@
 <?php
+/**
```

Comment:
Convert this entity, property type not defined, snake_case properties should changed to camelCase, define nullables